src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -125,6 +126,21 @@ internal class SourcePropertyAccessorSymbol : SourceMemberMethodSymbol
                 syntax,
                 diagnostics);
         }
+
+        // PROTOTYPE(semi-auto-props): Figure out what is going to be more efficient, to go after tokens and then
+        // checking their parent, or to go after nodes (IdentifierNameSyntax) first and then checking the underlying token.
+        // PROTOTYPE(semi-auto-props): A field identifier following a . or a -> cannot be a keyword.
```

Comment:
>PROTOTYPE(semi-auto-props): A field identifier following a . or a -> cannot be a keyword. [](http://example.com/codeflow?start=11&length=89)

I am assuming this PROTOTYPE comment is supposed to record a work item. I.e. it is supposed to record what should be done. The wording doesn't sound like that. Consider adjusting accordingly. For example, I would word it like that: "Filter out identifiers that syntactically cannot be keywords. For example those that follow a `.`, a `->` or a `::` in names. Something else?"

Reply:
Fixed